src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -182,6 +188,18 @@ private static ArgumentParser getArgumentParser() {
                         + "One kind of valid timezones is relative to UTC. E.g. UTC, UTC+08, UTC-1030. \n"
                         + "If not provided, system default timezone will be used.");
 
+        parser.addArgument(CLONING_THREADS_FLAG)
+                .dest(CLONING_THREADS_FLAG[0])
+                .type(int.class)
```

Comment:
Trivial, but I think we should create a `CloningThreadsArgumentType` instead of just using `int`.